lib/util/http/github.ts

```diff
@@ -210,31 +210,26 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
       if (opts.paginate) {
         // Check if result is paginated
         const pageLimit = opts.pageLimit ?? 10;
-        const linkHeader =
-          result?.headers?.link &&
-          parseLinkHeader(result.headers.link as string);
+        const linkHeader = is.string(result?.headers?.link)
+          ? parseLinkHeader(result.headers.link)
+          : null;
         if (linkHeader?.next && linkHeader?.last) {
           let lastPage = +linkHeader.last.page;
```

Comment:
I know it's not related to your changes, but it looked funky 😅 

```suggestion
          let lastPage = Number.parseInt(linkHeader.last.page);
```